config/packages/twig.yaml

```diff
@@ -2,4 +2,4 @@ twig:
     default_path: '%kernel.project_dir%/templates'
     debug: '%kernel.debug%'
     strict_variables: '%kernel.debug%'
-    form_themes: ['bootstrap_4_layout.html.twig']
+    form_themes: ['forms/custom_form_theme.html.twig']
```

Comment:
@daria-sieroshtan 
`custom_form_theme` is fine as an exercise, but in production you just say `form`

Reply:
I have "form" name already reserved for, hmm, form. And this is actually form theme.

Reply:
I have "form" name already reserved for, hmm, form. And this is actually form theme.

Reply:
I have form already. And this is form_theme.